eth/handler_eth_test.go

```diff
@@ -392,8 +380,8 @@ func testSendTransactions(t *testing.T, protocol uint) {
 
 // Tests that transactions get propagated to all attached peers, either via direct
 // broadcasts or via announcements/retrievals.
-func TestTransactionPropagation64(t *testing.T) { testTransactionPropagation(t, 64) }
 func TestTransactionPropagation65(t *testing.T) { testTransactionPropagation(t, 65) }
+func TestTransactionPropagation66(t *testing.T) { testTransactionPropagation(t, 66) }
```

Comment:
We really should use `eth.ETH6X` for these magic values..

Reply:
Myeah, that would keep us from forgetting them. I deliberately used constants way back to not link them to the live code (i.e. "better test independence"), but that seems stupid in retrospect.

Reply:
Fixed